packages/react-components/review/react-components.api.md

```diff
@@ -458,6 +458,22 @@ export interface ErrorBarStrings {
 // @public
 export type ErrorType = keyof ErrorBarStrings;
 
+// @beta
+export interface FileMetadata {
+    extension: string;
+    name: string;
+    url: string;
+}
+
+// @beta
+export interface FileUploadState {
+    errorMessage?: string;
+    filename: string;
+    id: string;
+    metadata?: FileMetadata;
```

Comment:
activeFileUploads doesn't need the `metadata` prop. You can omit it. Which means you won't have to re-create `FileMetadata` interface here.
This interface should be called `ActiveFileUploads` as to an end user `FileUploadState` has no meaning here.

Reply:
done